src/vm/ecalllist.h

```diff
@@ -373,10 +373,8 @@ FCFuncEnd()
 
 FCFuncStart(gTypedReferenceFuncs)
     FCFuncElement("InternalToObject", ReflectionInvocation::TypedReferenceToObject)
-#ifndef FEATURE_CORECLR
     FCFuncElement("InternalSetTypedReference", ReflectionInvocation::SetTypedReference)
```

Comment:
The TypedReference and FieldInfo ones will actually get called.

These ones will not [src](https://github.com/dotnet/coreclr/pull/9203/files#diff-3667dffbd11675529c85670ef344242eL1815) because these ones are ArgIterator functions.